include/zephyr/net/ieee802154_mgmt.h

```diff
@@ -42,14 +42,14 @@ enum net_request_ieee802154_cmd {
 	NET_REQUEST_IEEE802154_CMD_CANCEL_SCAN,
 	NET_REQUEST_IEEE802154_CMD_ASSOCIATE,
 	NET_REQUEST_IEEE802154_CMD_DISASSOCIATE,
-	NET_REQUEST_IEEE802154_CMD_SET_CHANNEL,
-	NET_REQUEST_IEEE802154_CMD_GET_CHANNEL,
-	NET_REQUEST_IEEE802154_CMD_SET_PAN_ID,
-	NET_REQUEST_IEEE802154_CMD_GET_PAN_ID,
-	NET_REQUEST_IEEE802154_CMD_SET_EXT_ADDR,
-	NET_REQUEST_IEEE802154_CMD_GET_EXT_ADDR,
-	NET_REQUEST_IEEE802154_CMD_SET_SHORT_ADDR,
-	NET_REQUEST_IEEE802154_CMD_GET_SHORT_ADDR,
+	NET_REQUEST_IEEE802154_CMD_SET_CHANNEL, /* in CPU byte order */
+	NET_REQUEST_IEEE802154_CMD_GET_CHANNEL, /* in CPU byte order */
+	NET_REQUEST_IEEE802154_CMD_SET_PAN_ID, /* in CPU byte order */
+	NET_REQUEST_IEEE802154_CMD_GET_PAN_ID, /* in CPU byte order */
+	NET_REQUEST_IEEE802154_CMD_SET_EXT_ADDR, /* in big endian byte order */
```

Comment:
The commit message in `net: l2: ieee802154: document and fix endianness` mentions "external address" twice - shouldn't it rather be "extended address"?

Reply:
Oups. Of course. This was just a test whether you actually read my commit messages. ;-)